libs/domain/cart/src/controllers/totals.controller.ts

```diff
@@ -30,7 +30,7 @@ export class TotalsController implements ReactiveController {
     return defer(() =>
       this.context.get<string>(TotalsContext.Reference).pipe(
         switchMap((context) => {
-          return this.totalsService.get(context);
+          return this.totalsService.get(context, { contextElement: this.host });
```

Comment:
1. What about having TotalService.get({ context,, element, ... }) for better upgradability in the future. 
2. What's better, passing element or cart id to the totals service? Just question. TotalService is mostly used inside component to solve totals from context, so passing element may be better indeed

Reply:
1. fixed
2. yup, I don't think that we will use this service outside totals components